src/components/ListItem/ListItem.tsx

```diff
@@ -65,7 +95,7 @@ export const ListItem: React.FC<IListItemProps> = ({
             </Typography>
 
             {
-            iconEnd &&
+                iconEnd &&
                 <iconEnd.type {...iconEnd.props} size="small" className={classnames('moonstone-listItem_iconEnd')}/>
```

Comment:
Same here the classname `moonstone-listItem_iconEnd` isn't display

Reply:
The className should appear now.